esphome/components/tuya/__init__.py

```diff
@@ -9,12 +10,17 @@
 Tuya = tuya_ns.class_('Tuya', cg.Component, uart.UARTDevice)
 
 CONF_TUYA_ID = 'tuya_id'
+CONF_CLOCK = 'clock'
 CONFIG_SCHEMA = cv.Schema({
     cv.GenerateID(): cv.declare_id(Tuya),
+    cv.Optional(CONF_CLOCK): cv.use_id(time.RealTimeClock),
```

Comment:
For consistency with other components that use `time` maybe use `CONF_TIME_ID` and `var.set_time(time_)`?
```suggestion
    cv.Optional(CONF_TIME_ID): cv.use_id(time.RealTimeClock),
```

Reply:
Should I rename `clock` to `time` in the code? Will it be more consistent with the codebase?

Reply:
Yeah, thanks =)